### PR TITLE
Add iHaveNoRowsInTheTable step.

### PR DESCRIPTION
**Pull request by sixcorners**
This contains exactly the code mentioned in #6. While I haven't tested this code in this project it works on my own MyStepdefs class.

### CODE REVIEW

File: src/main/java/org/tomitribe/beryllium/db/DatabaseSteps.java [hunk 1]
```diff
@@ -80,6 +80,11 @@ public void iHaveOnlyTheFollowingRowsInTheTable(final String tableName,
         this.insert(tableName, data);
     }
 
+    @Given("^I have no rows in the \"([^\"]*)\" table$")
+    public void iHaveNoRowsInTheTable(String tableName) throws Throwable {
```
Comment (cchacin) [inline src/main/java/org/tomitribe/beryllium/db/DatabaseSteps.java]: minor: use `final` in the parameter

**Comment by cchacin** [comment]
@sixcorners can you add a test for this? thanks

**Comment by cchacin** [comment]
LGTM, I'll merge this after you add the test

**Comment by sixcorners** [comment]
I installed mysql and added the schema "myapp_test". I installed redis. What else do I need to do to get unit tests passing?

**Comment by sixcorners** [comment]
There isn't an easy way to represent an empty DataTable in gherkin is there?

File: src/test/resources/features/successful-endpoints.feature [hunk 2]
```diff
@@ -123,6 +123,16 @@ Feature: Successful rest calls
       | Carlos2 |
     And response json path element "$[0].id" should be "1"
 
+  Scenario: Retrieve users list cleaning db
+    Given I have no rows in the "models" table
```
Comment (cchacin) [inline src/test/resources/features/successful-endpoints.feature]: I would add here something like:

``` gherkin
Given I have the following rows in the "models" table
| | | |
When I have no rows in the "models" table
Then I make a GET ....
And response should be json:
"""
[]
"""
```

to verified that we are actually removing the existing rows from the db

**Pull request merged**